src/Core/src/Platform/iOS/MauiUIApplicationDelegate.Menu.cs

```diff
@@ -1,4 +1,6 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
We try to stay away from using LINQ with any core code because it can occasionally lead to some performance issues. Can you rework the code to not use linq?